lib/public/views/lhcPeriods/Overview/LhcPeriodsOverviewModel.js

```diff
@@ -108,5 +138,47 @@ export class LhcPeriodsOverviewModel extends Observable {
     reset() {
         this._lhcPeriods = RemoteData.NotAsked();
         this._pagination.reset();
+        this._namesFilterModel.reset();
+        this.fetchLhcPeriods();
+    }
+
+    /**
+     * Register a new filter model
+     * @param {FilterModel} filterModel the filter model to register
+     * @return {void}
+     * @private
+     */
+    _registerFilter(filterModel) {
+        filterModel.visualChange$.bubbleTo(this);
+        filterModel.observe(() => {
+            if (filterModel.isValid) {
+                this.fetchLhcPeriods();
+            }
+        });
```

Comment:
The filter model should not notify if it is not valid

Reply:
right, refactored